Makefile

```diff
@@ -98,6 +98,11 @@ endif
 	# Diff of uncommitted changes for running locally
 	git diff HEAD -- rasa | poetry run flake8 --select D --diff
 
+lint-changelog:
+# Lint changelog filenames to avoid merging of incorrectly named changelog fragment files
+# For more info about proper changelog file naming, see https://github.com/RasaHQ/rasa/blob/main/changelog/README.md
```

Comment:
```suggestion
	# Lint changelog filenames to avoid merging of incorrectly named changelog fragment files
	# For more info about proper changelog file naming, see https://github.com/RasaHQ/rasa/blob/main/changelog/README.md
```